### ISSUE TITLE
App broken after entering wrong password

### ISSUE THREAD

**Issue by redm123** [nextcloud/talk-ios#306]
I wanted to try Nextcloud Talk. So I'm starting with fresh default settings.

And yea, no surprise with NC, nothing works..  First no QR code scan. So I tried regular password. I got the extremely helpful message "CSRF Check Failed". Thanks...  Ok, after that was solved I tried again. 

And I mistyped my password! OMG! The app still proceeded into something what looks like the app interface, empty with a spinning wheel in the center. And that's it. You can't try again. Even after killing and restartig the app you are in the same state. You can't log out.
Selected the obvious "Add account" option, which complained that the account would already exist. Broken, of course. 

Ok, tapping on the super intuitive empty-circle-name field, let me remove the broken account and try again. Which worked. Hey! 
Does somebody at NC test such obvious things? It's not that it would be the first obviously broken thing ...

Sorry for the rant...

Long story short: A failing login should of course not create a broken account.

**Comment by Ivansss** [comment]
Sorry but I can't reproduce your issue.
This is what you get when entering a wrong password:

![IMG_E0284](https://user-images.githubusercontent.com/4638605/80983878-6527c880-8e2d-11ea-8710-d7e3832afd3c.JPG)

> Sorry for the rant...

No worries, but ranting and sarcasm are just not helpful at all.
Information about how to reproduce your issue, the Nextcloud sever version and Talk iOS app version you are using...could be much more helpful 😉

**Comment by redm123** [comment]
Ok, it happens when you login via the app token path. Enter a name but no token and proceed.

Talk app: 8.1.0
NC server: 18.0.4

> No worries, but ranting and sarcasm are just not helpful at all.

I'm aware. Sorry, about that ;) I updated the server and tried the Talk functionality, and I hit 5-7 more or less grave beaking issues (e.g. just the presence of the Talk app let the NC server upgrade fail, since, I don't know, January), and for each one of them I just thought: Did anyone ever test this at all?? And that's often my experience with NC. I imaging other people with way less IT knowlege...no way. As much as I like the idea of NC, it's sometimes though... Anyway, getting off topic.

**Comment by Ivansss** [comment]
> Ok, it happens when you login via the app token path. Enter a name but no token and proceed.
> 
> Talk app: 8.1.0
> NC server: 18.0.4

That's a more valuable information :)
I can reproduce your issue now and I have just opened an [issue](https://github.com/nextcloud/server/issues/20838) in the Nextcloud server repository.
The login flow comes from the server so the bug will be fixed there.
